docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,131 @@
+undergoes a set of corresponding transformations and can contain additional operations required for the optimization. In
+the case of QAT, the compression controller object is used for model export and, optionally, in distributed training as it
+will be shown below.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [wrap_model]
+
+
+4. Fine-tune the model
+####################################
+
+This step assumes that you will apply fine-tuning to the model the same way as it is done for the baseline model. In the
+case of QAT, it is required to train the model for a few epochs with a small learning rate, for example, 10e-5. In principle,
+you can skip this step which means that the post-training optimization will be applied to the model.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [tune_model]
+
+
+5. Multi-GPU distributed training
+####################################
+In the case of distributed multi-GPU training (not DataParallel), you should call ``compression_ctrl.distributed()`` before
+the fine-tuning that will inform optimization methods to do some adjustments to function in the distributed mode.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [distributed]
+
+
+.. note::
+   The precision of weights gets INT8 only after the step of model conversion to OpenVINO Intermediate Representation.
+   You can expect the model footprint reduction only for that format.
+
+
+These were the basic steps to applying the QAT method from the NNCF. However, it is required in some cases to save/load model
+checkpoints during the training. Since NNCF wraps the original model with its own object it provides an API for these needs.
+
+6. (Optional) Save checkpoint
+####################################
+
+To save model checkpoint use the following API:
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [save_checkpoint]
+
+
+7. (Optional) Restore from checkpoint
+################################################
+
+To restore the model from checkpoint you should use the following API:
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [load_checkpoint]
+
+
+For more details on saving/loading checkpoints in the NNCF, see the following `documentation <https://github.com/openvinotoolkit/nncf/blob/develop/docs/Usage.md#saving-and-loading-compressed-models>`__.
+
+Deploying quantized model
+#########################
+
+The model can be converted into the OpenVINO Intermediate Representation (IR) if needed, compiled and run with OpenVINO.
+No extra steps or options are required.
```

Comment:
```suggestion
Deploying the Quantized Model
#################################

The model can be converted into the OpenVINO Intermediate Representation (IR) if needed, compiled, and run with OpenVINO without any additional steps.
```

Reply:
The same comment about `,` before the `and`